public/js/tracking_protection.js

```diff
@@ -27,13 +27,23 @@ function browserName() {
     return "unknown";
   }
 }
-  
-  if (browserName() === "firefox") {
-    if (document.getElementById("download-firefox")) {
-    document.getElementById("download-firefox").classList.add("hide");
-    }
+
+
+function removeUtms() {
+  const win = window;
+  const loc = window.location;
+  if (loc.href.indexOf("utm") !== -1 && win.history.replaceState) {
```

Comment:
Do we need the entire `href`? Or can we limit it to `loc.href.search` query string? Should we check for just "utm" or "utm_"? I guess my fear is that we'd have some breach name which just happens to coincidentally have "utm" string somewhere in the Name and we accidentally get into this flow.

Also, we could possibly use something like [`String#includes()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/includes), a la:

```js
if (loc.search.includes("utm_") && win.history.replaceState) {..}
```

Not sure if it'd be worth it to try parsing the entire search query into an object and verifying that the query parameter name starts with some `/^utm_/i` regexp.